src/screens/MenuScreen.tsx

```diff
@@ -20,6 +22,11 @@ const randRecipeUrl = `https://api.spoonacular.com/recipes/random?apiKey=${API_K
 
 export default function MenuScreen() {
     const [randRecipes, setRandRecipes] = React.useState<Recipe[]>([initialState.recipe]);
+    const user = useSelector<RootState, UserState>((state) => state.userState);
```

Comment:
Can we change 'user' to userState? I only suggest this because UserState has a property of 'user' and 'isLoggedIn'.

If we want user data we would have to use the syntax 'user.user', I think 'userState.user' might make more sense. Let me know your thoughts